loadsave.c

```diff
@@ -7,7 +7,9 @@
 #include <stdbool.h>
 #include <sys/stat.h>
 #include <dirent.h>
```

Comment:
This file does not exist on Visual Studio (2019).

Reply:
I added this alternative https://github.com/tronkko/dirent
I had it installed globally via vcpkg. Thanks for noting.